var/spack/repos/builtin/packages/visit/package.py

```diff
@@ -137,7 +138,8 @@ class Visit(CMakePackage):
 
     # VisIt uses Silo's 'ghost zone' data structures, which are only available
     # in v4.10+ releases: https://wci.llnl.gov/simulation/computer-codes/silo/releases/release-notes-4.10
-    depends_on("silo@4.10: +shared", when="+silo")
+    # Silo 4.11 doesn't compile with gcc 11.1.
```

Comment:
If this is a GCC conflict for Silo, then it should probably be expressed as a `conflicts("%gcc@11.1:", when="@4.11")` in the Silo package, not in the VisIt package. This way you can still build VisIt wth silo 4.11 when using compilers that do work.

Reply:
@kwryankrattiger got it. I'll submit a PR for that.